evergreen/config.yml

```diff
@@ -1546,10 +1563,25 @@ buildvariants:
     python3: "/opt/mongodbtoolchain/v3/bin/python3"
     use_system_openssl: On
     fetch_missing_dependencies: On
+    cmake_build_type: RelWithDebInfo
   tasks:
   - name: compile_test_and_package
   - name: benchmarks
 
+- name: ubuntu2204-arm64-asan
+  display_name: "Ubuntu 22.04 ARM64 (ASAN)"
+  run_on: ubuntu2204-arm64-large
+  expansions:
+    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.20.3-linux-aarch64.tar.gz"
+    cmake_bindir: "./cmake_binaries/bin"
+    python3: "/opt/mongodbtoolchain/v3/bin/python3"
+    use_system_openssl: On
+    fetch_missing_dependencies: On
+    cmake_build_type: Debug
+    enable_asan: On
+  tasks:
+  - name: compile_test
+
```

Comment:
So you got `ubuntu2204-arm64-asan` to pass by suppressing the messages around `adjtime` - nice!

Reply:
not quite, this is asan, not tsan. The problem with adjtime was a new one after #6911. It also fails builds from time to time. TSAN on ubuntu on arm64 still needs a lot of fixes. The test build on ubuntu2204-arm64-large showed a few hundred issues, i didn't yet look into them apart from linked open ones.